index.html

```diff
@@ -0,0 +1,193 @@
+              </div>
+              <p><a href="#" target="_blank">Read More</a></p>
+          </article>
+        </div>
+      </div>
+    </section>
+    <section class="latest-blog">
+      <div class="container">
+        <div class="head-title text-center">
+          <h2>Latest From Our Blog</h2>
+          <div class="separator"></div>
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut in egestas lectus.
+              Etiam tempor odio tellus, at bibendum neque faucibus quis. 
+              Sed vel facilisis elit. Nullam fringilla pharetra diam non accumsan. 
+              Morbi eget aliquam mauris. Etiam vehicula efficitur mi.
+          </p>
+         </div>
+        <div class="row">
+          <article class="grid-item col-lg-3 col-sm-6 col-xs-12">
+            <img src="images/img1.png" alt="img1">
+            <div class="content-text">
+              <h4>Project Title</h4>
+              <p>Portortornec condimenterdum eget consectetuer condis consequam pretium pellus sed mauris enim. Puruselit mauris nulla hendimentesque elit semper nam a sapien urna sempus.</p>
+            </div>
+            <a href="#" class="btn btn--primary">View Details</a>
+          </article>
+          <article class="grid-item col-lg-3 col-sm-6 col-xs-12">
+            <img src="images/img2.png" alt="img2">
+            <div class="content-text">
+              <h4>Project Title</h4>
+              <p>Portortornec condimenterdum eget consectetuer condis consequam pretium pellus sed mauris enim. Puruselit mauris nulla hendimentesque elit semper nam a sapien urna sempus.</p>
+            </div>
+            <a href="#" class="btn btn--primary">View Details</a>
+          </article>
+          <article class="grid-item col-lg-3 col-sm-6 col-xs-12">
+            <img src="images/img3.png" alt="i3g1">
+            <div class="content-text">
+              <h4>Project Title</h4>
+              <p>Portortornec condimenterdum eget consectetuer condis consequam pretium pellus sed mauris enim. Puruselit mauris nulla hendimentesque elit semper nam a sapien urna sempus.</p>
+            </div>
+            <a href="#" class="btn btn--primary">View Details</a>
+          </article>
+          <article class="grid-item col-lg-3 col-sm-6 col-xs-12">
+            <img src="images/img4.png" alt="img4">
+            <div class="content-text">
+              <h4>Project Title</h4>
+              <p>Portortornec condimenterdum eget consectetuer condis consequam pretium pellus sed mauris enim. Puruselit mauris nulla hendimentesque elit semper nam a sapien urna sempus.</p>
+            </div>
+            <a href="#" class="btn btn--primary">View Details</a>
+          </article>
+        </div>
+      </div>
+    </section>
+    <footer class="page-footer">
+      <div class="footer-info container">
+        <div class="row">
+          <section class="col-lg-3 col-xs-12">
+            <h4>CONTACT DETAILS</h4>
```

Comment:
Don't write uppercase content in HTML, only write capitalize for the first letter and add style to it.